lib/appsignal/logger/backend.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Appsignal.Logger.Backend do
+  @behaviour :gen_event
+
+  def init({__MODULE__, group}) do
```

Comment:
In the Node.js instrumentation, we used positional arguments to set the group, and now when adding the format the signature has become a bit unwieldy. Could we use an options list here, to get ahead of that for future extensibility?

(It also makes it clearer what the "phoenix" in `{Appsignal.Logger.Backend, "phoenix"}` means!)

```suggestion
  def init({__MODULE__, [groupname: group]}) do
```